pkgs/top-level/all-packages.nix

```diff
@@ -8155,11 +8155,7 @@ in
 
   neko = callPackage ../development/compilers/neko { };
 
-  nextpnr = libsForQt5.callPackage ../development/compilers/nextpnr {
-    # QT 5.12 has a weird regression involving the floorplanning window having
-    # a 'blank' or 'transparent' background, so fall back to 5.11 for now.
-    qtbase = qt511.qtbase;
```

Comment:
note: The proper way to have done that would have been to use `libsForQt511.callPackage`